### ISSUE TITLE
feedback

### ISSUE THREAD

**Issue by defunctzombie** [duojs/duo#4]
Don't know where else to put the feedback so I opened this issue.

TL;DR; you are not fixing enough real problems over browserify/npm/github-url to warrant yet another package manager/bundler/pipeline/etc

I think tying your registry and package installation tool to your asset bundler is a mistake. It was a mistake in component and I think it is a mistake in whatever duo is trying to be. Component became a `god` tool that does everything and as a result imo became shitty at anything because maintaining such a tool is no small undertaking. More importantly, asset packaging, installation, dependencies, etc is all very opinionated and by bundling everything into a single tool then you drastically limit experimentation or access.

I have more opinions on this but really I don't see enough wins to warrant switching and many of the items listed as "bad" have trivial workarounds.
